aspect/intellij_info_impl.bzl

```diff
@@ -513,16 +521,27 @@ def collect_cpp_info(target, ctx, semantics, ide_info, ide_info_file, output_gro
     target_copts = _do_starlark_string_expansion(ctx, "copt", target_copts, extra_targets)
 
     compilation_context = target[CcInfo].compilation_context
+    defines = compilation_context.defines.to_list()
+    includes = compilation_context.includes.to_list()
+    quote_includes = compilation_context.quote_includes.to_list()
+    system_includes = compilation_context.system_includes.to_list()
+
+    if hasattr(ctx.rule.attr, "implementation_deps"):
```

Comment:
can't this information be obtained from `CcInfo` provider?

Reply:
I don't think there is a way since bazel treats impl deps context different from the main context. CcInfo only exposes the main context as far as I can tell. 

https://github.com/bazelbuild/bazel/blob/b85db78de3b25cb7f5c9cb0c7afe388f75e040e3/src/main/starlark/builtins_bzl/common/cc/cc_compilation_helper.bzl#L449